db/schema.rb

```diff
@@ -12,416 +12,416 @@
 
 ActiveRecord::Schema.define(version: 20170602231912) do
 
-  create_table "alerts", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
-    t.integer  "course_id"
-    t.integer  "user_id"
-    t.integer  "article_id"
-    t.integer  "revision_id"
-    t.string   "type"
+  create_table "alerts", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
I think this is unrelated to the rest, and just got included by accident. Shouldn't be any schema changes.